sampleAppKotlin/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,21 @@
             </intent-filter>
         </activity>
         <activity android:name=".ProductActivity" />
+        <activity
```

Comment:
Client has to include this into their AndroidMainfest.xml to enable deep linking from Custom Custom Tabs back into the app